src/components/item/ItemContent.tsx

```diff
@@ -268,17 +269,28 @@ const AppContent = ({
   <AppItem
     isResizable
     item={item}
-    apiHost={API_HOST}
     editCaption={isEditing}
     onSaveCaption={onSaveCaption}
     onCancelCaption={onCancelCaption}
     saveButtonId={saveButtonId}
     cancelButtonId={cancelButtonId}
-    member={member}
     height={ITEM_DEFAULT_HEIGHT}
-    permission={permission}
-    requestApiAccessToken={Api.requestApiAccessToken}
-    context={CONTEXT_BUILDER}
+    requestApiAccessToken={(payload) =>
+      Api.requestApiAccessToken(payload, { API_HOST })
+    }
+    contextPayload={{
+      apiHost: API_HOST,
```

Comment:
Now `API_HOST` has to be passed twice, I am not sure this simplified the interface